go/vt/vtctl/vtctl.go

```diff
@@ -2275,6 +2276,8 @@ func commandGetSchema(ctx context.Context, wr *wrangler.Wrangler, subFlags *flag
 	excludeTables := subFlags.String("exclude_tables", "", "Specifies a comma-separated list of tables to exclude. Each is either an exact match, or a regular expression of the form /regexp/")
 	includeViews := subFlags.Bool("include-views", false, "Includes views in the output")
 	tableNamesOnly := subFlags.Bool("table_names_only", false, "Only displays table names that match")
+	tableSizesOnly := subFlags.Bool("table_sizes_only", false, "Only displays size information for tables")
```

Comment:
I think we need to handle the case when both `tableNamesOnly` and `tableSizesOnly` are specified. The logic should  clear about what is returned.  Now, it looks like only table names are returned.